test/DynamoCoreWpfTests/ViewExtensions/ViewExtensionTests.cs

```diff
@@ -346,6 +353,44 @@ public static void RaiseLoadedEvent(FrameworkElement element)
 
             eventMethod.Invoke(element, new object[] { args });
         }
+
+        /// <summary>
+        /// Test if the number of nodes displayed in the extension is equal to current number of nodes
+        /// </summary>
+        [Test]
+        public void TestGraphEvaluationEvents()
+        {
+            RaiseLoadedEvent(this.View);
+            var extensionManager = View.viewExtensionManager;
+            extensionManager.Add(viewExtension);
+
+            EngineController controller = null;
+            int counter = 0;
+            viewExtension.OnOpenEvaluationStarted += (HomeWorkspaceModel hwm, EventArgs args) =>
```

Comment:
unsub these somewhere in the test or in a finally etc.

Reply:
I am not sure if we need to. Aren't all these objects destroyed anyway at the end of the test?

Reply:
I don't know - it's hard to prove without using a memory profiler.

I guess you can run the master-15 self service and see if the job runs out of memory or if the test time goes up - but it seems faster to just always unsubscribe.

Reply:
done